google-cloud-graalvm-samples/spring-cloud-gcp-trace-sample/src/main/resources/META-INF/native-image/com.example/proxy-config.json

```diff
@@ -0,0 +1,3 @@
+[
+    ["com.example.WorkService", "org.springframework.aop.SpringProxy", "org.springframework.aop.framework.Advised", "org.springframework.core.DecoratingProxy"]
```

Comment:
Right, so this has to do with something in Spring where sometimes their dependency injection requires declaring a proxy like this. I think Spring team has plans to "autodetect" usages like these, but this not something related to GCP; I think it's okay not to document here since users of Spring + GraalVM will probably be familiar with having to do things like this.